### PR TITLE
Fixes #7526 Added icons to Multimedia menu in Note Editor

### LINKED ISSUE #7526: Note Editor: Add Icons to Multimedia Menu
Good first issue - need to replace icons with XML drawables (Via Add - Vector Icon), then set a style/tint on the XML icons.

https://github.com/ankidroid/Anki-Android/blob/71fdd0445f020ec7f798e29bc08dc13d658d9234/AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.java#L1405-L1406

Menu currently looks like:

image

### PR DESCRIPTION
**Pull request by mrudultora**
## Purpose / Description
Added icons to Multimedia menu in Note Editor. (working for all the themes)

## Fixes
Fixes #7526 

## Approach
Added new vector icons and then used them in Popup Multimedia Menu.
**After the change the menu now looks like this.**

img

## How Has This Been Tested?
Tested on real device (Redmi Note 7S , API 29) and on emulator also (Pixel 3a , API 21).

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml [hunk 1]
```diff
@@ -1,23 +1,41 @@
...
-        android:title="@string/multimedia_editor_attach_media" >
+    <group
+        android:id="@+id/menu_multimedia_group"
+        android:title="@string/multimedia_editor_attach_media">
         <item
             android:id="@+id/menu_multimedia_photo"
-            android:title="@string/multimedia_editor_popup_image"/>
+            android:icon="@drawable/ic_popup_menu_item_image"
+            android:title="@string/multimedia_editor_popup_image"
+            app:showAsAction="always"
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml:13]: Could you explain (possibly in a comment on the file) why `app:showAsAction="always"` is required

File: AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml [hunk 2]
```diff
@@ -1,23 +1,41 @@
...
+            tools:ignore="AlwaysShowAction" />
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml:14]: It might be worth moving this to an "ignore" for the file, to remove line noise

**Review by david-allison-1 — COMMENTED** [review]
Looks fantastic. One minor request, and good to go. Thanks!!!

File: AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml (hunk 1 above)
Comment (mrudultora) [reply AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml:13]: I tried without it, was not working. Android documentation also does not explain this. It was suggested in **SO** thread.

File: AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml (hunk 2 above)
Comment (mrudultora) [reply AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml:14]: Okay, I will do that. Actually, it was giving warning. So, I just did this. I will remove this.

Comment (david-allison-1) [reply AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml:14]: You (should?) be able to move `tools:ignore="AlwaysShowAction"` into the XML declaration at the top of the file - we want to get rid of warnings if they're not relevant, but having the repeated declaration makes reading the code slightly harder.

File: AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml (hunk 1 above)
Comment (david-allison-1) [reply AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml:13]: Sounds good to me - just add an explanation & link to the SO thread so others don't have the same questions

File: AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml (hunk 2 above)
Comment (mrudultora) [reply AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml:14]: Ohh, I got that now. I interpreted in a wrong way.

**Comment by mrudultora** [comment]
@david-allison-1 I tried removing `showAsAction=true` and cleaned and rebuilded the project. It's working without that also. So, basically I have removed all the impacted lines.

**Review by david-allison-1 — APPROVED** [review]
Thanks again. Pending squash & merge after CI passes

**Review by mikehardy — APPROVED** [review]
Nice!

**Comment by mikehardy** [comment]
Hi there!

Just a friendly notice (one per person for PRs merged in March, regardless of PR quantity) that we try to process OpenCollective payments monthly - it's time for March 2021 submissions

If you are interested in compensation for this work, the process with details is here:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid

Thanks!

**Pull request merged**